content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Node.js
+title: TypeScript/ Node.js
 aliases: [/docs/js/getting_started/nodejs]
 ---
 
-This guide will show you how to get started with tracing in Node.js.
+This guide will show you how to get started with tracing in TypeScript and Node.js.
 
 ## Example Application
```

Comment:
Maybe we need a **Prerequisites** section before this section too.

Reply:
Cool! I have added same statement as one suggested for `browser.md`